spec/requests/api/v1/people_controller_index_spec.rb

```diff
@@ -10,42 +10,39 @@
 
   describe 'GET /people' do
     let(:prison_number) { 'G5033UT' }
+    let(:params) { { filter: { police_national_computer: 'AB/1234567' }, access_token: token.token } }
 
-    describe 'filtering the results' do
```

Comment:
All requests have to include the filter so I've removed this outer block without changing any of the inner block contents.

Reply:
Review without whitespace changes by using https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/514/files?diff=split&w=1